lua/Network.lua

```diff
@@ -0,0 +1,264 @@
+-------------------------------------------------------------------------------------------
+-- TerraME - a software platform for multiple scale spatially-explicit dynamic modeling.
+-- Copyright (C) 2001-2016 INPE and TerraLAB/UFOP -- www.terrame.org
+
+-- This code is part of the TerraME framework.
+-- This framework is free software; you can redistribute it and/or
+-- modify it under the terms of the GNU Lesser General Public
+-- License as published by the Free Software Foundation; either
+-- version 2.1 of the License, or (at your option) any later version.
+
+-- You should have received a copy of the GNU Lesser General Public
+-- License along with this library.
+
+-- The authors reassure the license terms regarding the warranties.
+-- They specifically disclaim any warranties, including, but not limited to,
+-- the implied warranties of merchantability and fitness for a particular purpose.
+-- The framework provided hereunder is on an "as is" basis, and the authors have no
+-- obligation to provide maintenance, support, updates, enhancements, or modifications.
+-- In no event shall INPE and TerraLAB / UFOP be held liable to any party for direct,
+-- indirect, special, incidental, or consequential damages arising out of the use
+-- of this software and its documentation.
+--
+-------------------------------------------------------------------------------------------
+local terralib = getPackage("terralib")
+local binding = _Gtme.terralib_mod_binding_lua
+local tl = terralib.TerraLib{}
+
+local function getBELine(cell)
+	local geometry = tl:castGeomToSubtype(cell.geom:getGeometryN(0))
+	local nPoint = geometry:getNPoints()
+	local endPoint
+	local beginPoint = binding.te.gm.Point(geometry:getX(0), geometry:getY(0), geometry:getSRID())
+
+	for i = 1, nPoint - 1 do
```

Comment:
It seems that this `for` is unnecessary. You only need to get the `geometry:getX(nPoint)`, isn't it?